appveyor.yml

```diff
@@ -37,5 +37,5 @@ test_script:
 
 deploy_script:
   - IF "%APPVEYOR_REPO_TAG%" == "true" (npm run upload-win)
-  - IF "%APPVEYOR_REPO_TAG%" == "true" && "%nodejs_version%" == "8" (npm run upload-electron --build_v8_with_gn=false)
+  - IF "%APPVEYOR_REPO_TAG%" == "true" (if "%nodejs_version%" == "8" (npm run upload-electron --build_v8_with_gn=false))
```

Comment:
Super nit: can we make `if` uppercase, to match the others?

Reply:
Oh yes, of course – done!